src/test/Number.test.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react'
+import renderer from 'react-test-renderer'
+import {shallow} from 'enzyme'
+
+import Number from '../components/widgets/Number'
+
+describe('Number widget component', () => {
+  const mainStat = '125'
+  const goal = '250'
+  const progress = ((mainStat / goal) * 100).toString().concat('%')
+
+  const widgetData = {
+        type: 'Number',
+        title: 'Number',
+        size: '1x1',
+        x: '10px',
+        y: '500px',
+        mainStat: mainStat,
+        goal: goal
+      }
+
+  const numberWrapper = shallow( <Number {...widgetData} /> )
+
+  it('renders without crashing', () => {
+    const numberSnapshot = renderer.create( <Number {...widgetData} /> ).toJSON()
+    expect(numberSnapshot).toMatchSnapshot()
```

Comment:
Please add spaces between opening and closing parentheses, and arguments, in a function invocation or control flow expression:
```js
fn(param1, param2) { /* ... */ }
if(expression) { /* ... */ }
```
should be
```js
fn( params1, param2 ) { /* ... */ }
if( expression ) { /* ... */ }
```